apps/frontend/src/app/components/misc/apiHandler/APIHandler.tsx

```diff
@@ -0,0 +1,31 @@
+import { Heading, Loader } from '@navikt/ds-react';
+import { AxiosError } from 'axios';
+import { ReactElement } from 'react';
+
+interface IProps {
+    loading: boolean;
+    error: AxiosError | undefined;
+}
+
+export const APIHandler = ({ loading, error }: IProps): ReactElement => {
+    let element = <></>;
+
+    if (loading) element = <Loader />;
+
+    if (error) {
+        const errorStatus = error?.request.status;
+
+        switch (errorStatus) {
+            case 404:
+                element = <Heading size="medium">Ingen resultater funnet.</Heading>;
+                break;
+            case 403:
+                element = <Heading size="medium">Du har ikke tilgang til denne siden.</Heading>;
+                break;
+            default:
+                element = <Heading size="medium">Det har skjedd en feil...</Heading>;
+        }
+    }
```

Comment:
This is fine for now, but i think this switch case can get quite large as more error codes are added. Maby it can be changed so that every error code between 200-299 returns "Success" to the user etc

Reply:
Yup, it is definitely the intention that this should be altered as after our needs.